### ISSUE TITLE
Missing makefile target

### ISSUE THREAD

**Issue by brianredbeard** [carlos-algms/vscode-make-task-provider#4]
When attempting to use the extension with the following [Makefile](https://github.com/brianredbeard/acidwarp/blob/5fe0276018355a4da07d7d9d914e58a538ad8ba9/Makefile) the extension is missing the primary target (`acidwarp`):

`Makefile`:
```make
CC=gcc
CFLAGS = -O2 -funroll-all-loops
LFLAGS = -lvgagl -lvga
PROGS = acidwarp 
OSOURCES = bit_map.o palinit.o rolnfade.o lut.o

acidwarp: $(OSOURCES)
	$(CC) acidwarp.c $(CFLAGS) $(LFLAGS) $(OSOURCES) -o acidwarp ; strip acidwarp
bit_map.o: 
palinit.o:
rolnfade.o:
lut.o:
clean:
	rm -f acidwarp *.o
```

![image](https://user-images.githubusercontent.com/57542/106839805-c813ed80-6653-11eb-8310-6c7175540754.png)

**Comment by brianredbeard** [comment]
Interestingly: if I remove the variable definition of `OSOURCES` **or** the dependencies _from_ the `acidwarp` target (removing `$(OSOURCES)`), then the target `acidwarp` appears in the "Makefile tasks" list.

**Comment by brianredbeard** [comment]
Also, affected by #1

**Comment by carlos-algms** [comment]
Hello @brianredbeard, Thanks for taking your time to report this issue. 
It is indeed a Bug, a patch will be released soon. 

Basically, I could not find a reliable way of getting all available targets cross-platform. 
The REGEX parsing the output from `make ... --just-print` was not expecting something after the `:` delimiter.

**Comment by carlos-algms** [comment]
@brianredbeard I just released the patch in the `v1.4.1` and it should work on your case.
I used your `Makefile` to test and it is working. 

Please let me know if it is working for you so we can close this.

**Comment by brianredbeard** [comment]
Looking good. Thanks so much for the time invested in both building the provider and even more so the time spent improving it!
